pkgs/applications/networking/instant-messengers/telegram/tdesktop/default.nix

```diff
@@ -64,6 +64,7 @@ in mkDerivation rec {
     "-DTDESKTOP_API_HASH=d524b414d21f4d37f08684c1df41ac9c"
     #"-DDESKTOP_APP_SPECIAL_TARGET=\"\"" # TODO: Error when set to "": Bad special target '""'
     "-DTDESKTOP_LAUNCHER_BASENAME=telegramdesktop" # Note: This is the default
+    "-DDESKTOP_APP_DISABLE_GTK_INTEGRATION=ON"
```

Comment:
We are having trouble building with GTK_INTEGRATION on, namely `error: template with C linkage`, anyway what is GTK integration?